src/test/java/no/unit/nva/model/RoleDtoTest.java

```diff
@@ -52,18 +52,6 @@ public void toStringReturnsStringContainingTheNameOfTheRole() throws InvalidRole
         assertThat(role.toString(), containsString(role.getRoleName()));
     }
 
-    @Test
-    public void unexpectedExceptionThrowsRuntimeException() throws NoSuchMethodException {
```

Comment:
UnexpectedException (which was a checked exception)  was replaced by a RuntimeException and therefore we do not need to test the handling of throwing an UnexpectedException